scripts/run-sql.ts

```diff
@@ -1,6 +1,6 @@
-import { createConnection } from 'typeorm';
+import { Connection, createConnection } from 'typeorm';
```

Comment:
I didn't realize we had `typeorm` in this project.

My preference would be to use sequlize, since that is already a dependency of SQ and I don't want to risk bad interactions between ORMs (they might be OK together, but might not).

If you want to use typeorm, then you can use its migration features to change tables at least?